contracts/FROST.sol

```diff
@@ -266,8 +277,12 @@ library FROST {
         // to a curve scalar, there are some values of `z` that can be
         // specified in more than one way.
         {
-            bool pOk = _isOnCurve(px, py);
+            // px must be in range [1, _N) because of ecrecover requirements
+            // px != 0 because of _isOnCurve(px, py) is false if px == 0
+            // (there is no point on the curve with x == 0)
```

Comment:
It would be nice if this comment is instead a NatSpec comment in the `@dev` section on the `isValidPublicKey` function.

Reply:
661df0a1bf7599a34b039ca5f34c7037f05a2b57